dev-requirements.txt

```diff
@@ -1,2 +1,4 @@
 -r requirements.txt
 -r test-requirements.txt
+-r docs-requirements.txt
+sphinx-autobuild==0.7.1  # reload docs on changes
```

Comment:
I put this here in `dev-requirements.txt` because it's not required for building docs itself, but more of a dev environment setup.